libs/bootstrap-desktop.sh

```diff
@@ -22,36 +22,6 @@ else
     ./build-all.sh desktop
     popd
   fi;
-
-  # NSS system libs check.
```

Comment:
nice!

Reply:
For readers: this check is removed because we don't link dynamically to NSS but statically to the libs in `NSS_DIR`.